javascript/javascript-core/javascript-arrays/setting-the-length-of-an-array.md

```diff
@@ -66,9 +71,9 @@ arr.length = 4;
 ```
 
 - length
-- `[1,2,undefined,undefined]`
+- `[1,2,<2 empty items>]`
 - size
-- `[1,2,undefined,undefined,undefined]`
+- `[1,2,<3 empty items>]`
```

Comment:
Maybe leave this one as `undefined`

```suggestion
- `[1, 2, undefined, undefined]`
- `[1, 2, <1 empty item>, <1 empty item>]`
```

Reply:
[Undefined, undefined] or [<1empty sitem> , <1 empty item>]

Reply:
Aha, nvm I think I got it. The first correct answer should be:
- `[1, 2, <1 empty item>, <1 empty item>]`

and the incorrect one should be 

- `[1, 2, undefined, undefined]`

right?